content/blogposts/2022/release-3.42.md

```diff
@@ -0,0 +1,73 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/37984
+    category: Admin
+  - description: 'Gitserver endpoint access logs can now be enabled for added visbility by adding "log": { "gitserver.accessLogs": true } to the config.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/
+    category: Admin
+---
+
+Sourcegraph 3.42 is now available! For this release, we introduced:
+
+## All-new admin analytics
+
+We believe providing a world-class admin-experience is critical to the future of Sourcegraph customers around the world. This starts with empowering administrators to measure and track the value of Sourcegraph within their organization.
+
+Historically, Sourcegraph administrators rely on word-of-mouth feedback from developers, leading to a poor understanding of the value of Sourcegraph within an organization. 
+
+In 3.42, we are releasing a completely redesigned user analytics experience for administrators. These visualizations will make it easier to understand developer engagement, identify power users, and convey value to their internal leaders. 
+
+![Admin analytics page](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.42/admin_analytics.png)
+
+## New search UI, now in Beta
+
+We're fixing UX papercuts and working to improve the core search experience. 3.42 includes a new, optional Beta UI, which can be enabled via a toggle in the avatar menu. It's disabled by default but we encourage you to give it a try. We think you'll find it cleaner and easier to use. You can find the toggle just below the theme dropdown in the avatar menu.
+![](https://storage.googleapis.com/sourcegraph-assets/docs/images/code_search/beta-ui.gif)
+
+## Search by repository descriptions
+
+We've introduced a new search predicate that enables searching by repository description, for example: `repo:has.description(machine learning)`. Now you can search for interesting repository topics, even if you don't know their names.
+
+This is especially useful for engineers joining new teams or onboarding to a codebase at a new company, so you can quickly find repositories without knowing exactly what you're looking for.
+![](https://storage.googleapis.com/sourcegraph-assets/docs/images/code_search/repo-description-search.png)
+
+## Smart search suggestions with lucky search
+
+We have a new search type available experimentally, called "Lucky search". It improves the quality of your search experience by giving you smart suggestions for alternate queries, and it even runs them automatically to make sure they have results before recommending them. You can enable it by setting `"search.defaultPatternType": "lucky"` in settings.
+![Search suggestion example](https://storage.googleapis.com/sourcegraph-assets/docs/images/code_search/lucky-search.png)
+
+## Code insights scoped to explicit repository lists load faster and include more historical datapoints
```

Comment:
Should Code Insights be capitalized?

Reply:
@Joelkw @iskyOS - is this accurate:
Code Insights now load faster and include more historical data points

Is it necessary to say that it's scoped to explicit repository lists in the headline?

Reply:
@abschu @Joelkw I think that change is fine, and the details can be in the body of the highlight.

Reply:
```suggestion
## Insights load faster and include more historical datapoints
```